### ISSUE TITLE
Extend Google IdP to allow for parameters on the authorize URL

### ISSUE THREAD

**Issue by mooreds** [FusionAuth/fusionauth-issues#1852]
### Problem

As documented here: https://github.com/FusionAuth/fusionauth-site/pull/1563 there are time when you must append certain parameters to the Google authorize URL to allow for certain behavior, such as when prompting the user to pick from one of many google accounts.

### Solution

Would be great to have a way to add this to the Google Identity Provider.

Note that it should be overrrideable as you might want different behavior with different applications.

### Alternatives/workarounds

Use an OIDC provider without the discovery endpoints, as documented here: https://fusionauth.io/docs/v1/tech/identity-providers/google#custom-parameters

### Community guidelines
All issues filed in this repository must abide by the [FusionAuth community guidelines](https://fusionauth.io/community/forum/topic/1000/code-of-conduct).

### How to vote
Please give us a thumbs up or thumbs down as a reaction to help us prioritize this feature. Feel free to comment if you have a particular need or comment on how this feature should work.

**Comment by robotdan** [comment]
Is using the OIDC alternative reasonable enough such that we can close this? The purpose of the Google IdP is to make it super simple, and the OIDC should cover everything else.

**Comment by mooreds** [comment]
I would like to keep this open to see if the community wants this feature.

**Comment by epbensimpson** [comment]
Yes please. Even better would be to extend the Google provider so that the `prompt` parameter is supported directly.
